wp-content/themes/twentytwenty-child/functions.php

```diff
@@ -110,7 +129,9 @@ function get_student_info( $post_id ) {
     return $data;
 }
 
-/* Callback function with the HTML box for the custom settings */
+/** 
+ * Callback function with the HTML box for the custom settings 
+ * */
 function student_custom_box_html( $post ) {
     $data = array();
     if( ! empty( get_the_ID() ) ) $data = get_student_info( get_the_ID() );
```

Comment:
Try not to use `get_the_...` or `the_...` functions more that once. Every time you call such a function it queries the database.
A better way would be to sore the ID inside a variable and use the variable instead.